### ISSUE TITLE
Installation instructions don't work

### ISSUE THREAD

**Issue by ColinH** [flonatel/pipexec#9]
The "Installation - From Source" instructions don't work, they refer to a `configure` script that doesn't seem to be included. The `init_autotools.sh` script (that is not mentioned in the documentation) didn't work for me, either, it failed with some errors that I am not familiar with. I did manage to build `peet` with a simple `gcc -O3 -Wall src/version.c src/peet.c -I. -o peet` so the issue seems to be with the build system, not the source itself.

**Comment by flonatel** [comment]
Hi Colin :-)

Can you please check that you are really using the latest release tar ball - as described in the instructions? (I double checked it and found the configure script...) The install instructions are for somebody who 'just' wants to pick the latest release, compile and use the program.
When you 'git clone' the repo, there is no configure script. The init_autotools.sh is only needed when you clone the sources and either want to build the tar file or want to develop for pipexec.

Would it be possible to append the (important part of the) error when running init_autotools.sh?  In theory it should work - nevertheless there are some (currently undocumented) dependencies to autoconf, automake, libtool and so on.

**Comment by ColinH** [comment]
Hi Andre!

Ah, so the instructions in README.md say "Download the tar ball", but since I couldn't find one I took the ZIP of the repository instead...

On the bright side I now got `init_autotools.sh` running, there was something wrong with my installation that I wasn't aware of.

Now there are several hundred megabytes of scripts to invoke something like `gcc -O3 -Wall src/version.c src/peet.c -I. -o peet` :-)
